### PR TITLE
Maint/26 change logger

### LINKED ISSUE #26: Change Logger
**Is your feature request related to a problem? Please describe.**
At the moment the logger for deployment is only activated if the spring profile is dev, kon or prod. 

**Describe the solution you'd like**
Change the logger for deployments to be the default logger, without the need to add a spring profile.
Remove unnecessary loggers ("de.muenchen.allgemein is probably not needed anymore).
Introduce new profile `json-logging` to switch from normal logging format to json.

### PR DESCRIPTION
**Pull request by simonhir**
**Description**

Refactor logback spring profiles and cleanup:
- Default logging without profile
- New profile `json-logging`
- Cleanup

**Reference**

Issues closes #26

### CODE REVIEW

File: refarch-backend/src/main/resources/logback-spring.xml [hunk 1]
```diff
@@ -8,14 +8,16 @@
     <springProperty scope="context" name="springAppName" source="spring.application.name"/>
 
 
-    <springProfile name="local,test">
+    <springProfile name="!json-logging">
 
         <!-- Log appender -->
         <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
             <encoder>
+                <!-- @formatter:off -->
```
Comment (devtobi) [inline refarch-backend/src/main/resources/logback-spring.xml:16]: Which formatter gets disabled by this line? Is it the IDE internal formatting or Spotless?

**Review by devtobi — CHANGES_REQUESTED** [review]
Could you please provide those changes also for the refarch-eai template? :)

File: refarch-backend/src/main/resources/logback-spring.xml (hunk 1 above)
Comment (simonhir) [reply refarch-backend/src/main/resources/logback-spring.xml:16]: At least IDE. Not really sure if Spotless too.
Is needed as IDE is trying to wrap.

Comment (devtobi) [reply refarch-backend/src/main/resources/logback-spring.xml:16]: Fine. :)

**Review by devtobi — APPROVED** [review]
LGTM

**Review by devtobi — CHANGES_REQUESTED** [review]
Please apply the changes for the EAI-template as well. :)

**Review by devtobi — APPROVED** [review]
LGTM

**Review by DanielOber — APPROVED** [review]
LGTM

**Pull request merged**